src/hacbs/components/IntegrationTestsListView/IntegrationTestsListView.tsx

```diff
@@ -157,24 +140,6 @@ const IntegrationTestsListView: React.FC<IntegrationTestsListViewProps> = ({ app
             </Text>
           </TextContent>
         </FlexItem>
-        <FlexItem>
-          <ExternalLink
-            href="#"
-            text={
-              <Flex
-                spaceItems={{ default: 'spaceItemsXs' }}
-                justifyContent={{ default: 'justifyContentCenter' }}
-              >
-                <FlexItem className="pf-u-font-size-sm">
-                  Learn more about setting up an integration test
-                </FlexItem>
-                <FlexItem>
-                  <ExternalLinkAltIcon />
-                </FlexItem>
-              </Flex>
-            }
-          />
-        </FlexItem>
```

Comment:
nit: removing this flex item leaves the parent flex layout with only a single child. Seems as if we could just remove the flex layout altogether now.